Material.Blazor/Foundation/InputComponent.cs

```diff
@@ -347,22 +316,6 @@ protected sealed override bool ShouldRender()
         }
 
 
-        /// <summary>
-        /// Material.Blazor components descending from <see cref="InputComponent{T}"/> _*must not*_ override OnAfterRenderAsync(bool).
-        /// </summary>
-        protected override async Task OnAfterRenderAsync(bool firstRender)
```

Comment:
Obsolete now: no more need for delaying the initialization. This caused problems anyway, because this was the _only_ interop call we delayed, all other ones weren't.